### DISCUSSION TITLE
Сhat message in Ticked do not reach the user

### DISCUSSION THREAD

**Discussion by Andr198** [YetiForceCompany/YetiForceCRM#16333] [Feature improvement]
# 🐞 bug report

### ⁉️ Describe the bug

When adding a chat message in Ticked with a user tag messages do not reach the user

### 🔥 How to trigger the error

1. Go to 'Ticked-Chat'
2. Click on 'user tag '
3. Add user 'YetiForce Guest' and  Test
4. Exit and login for user YetiForce Guest after 5 min
![yeti1](https://user-images.githubusercontent.com/62097187/105368410-68e8be80-5bf9-11eb-8b83-5716ac136ce1.JPG)

### 👎 Actual Behavior

User did not receive message
![yeti2](https://user-images.githubusercontent.com/62097187/105369468-7d798680-5bfa-11eb-9f1c-bc9997a6efd3.JPG)

### 👍 Expected Behavior

Мessage to user

### 👷 Testing

- [x] I confirm that I tested it at https://gitstable.yetiforce.com
- [x] I confirm that I tested it at https://gitdeveloper.yetiforce.com

**Comment by deepti4666** [comment]
I have the same problem, even when trying to tag people in the comments, the user (tagged person) doesn't get the notification

**Comment by konadrian** [comment]
The current version doesn't have this functionality. We'll add it to the wishlist. The ticket has been closed
